cmd/util/cluster.go

```diff
@@ -158,6 +158,7 @@ type ClusterOptions struct {
 	ExecProviderAPIVersion  string
 	ExecProviderInstallHint string
 	ClusterEndpoint         string
+	DisableCompression      bool
```

Comment:
You added variable, but you are not using it. 

Take a look here 

https://github.com/argoproj/argo-cd/pull/20291/files#diff-1a32760cee151b5181abcd273fb66e286a02fd8c99573d988e423528760cc293R3194

This is how it should be used. Sorry if i reviewed it too early, maybe you just didnt finish your work